ivy/functional/frontends/torch/non_linear_activation_functions.py

```diff
@@ -21,7 +28,10 @@ def _compute_elu(input, alpha=1.0, inplace=False):
         return input
     return ret
 
-
+def sigmoid(input, out=None):
```

Comment:
If you're only adding `relu` activation function in the frontend, why have you also added the `sigmoid` function?

Reply:
Could you please take a look at it?
actually, Once the conflict occurred, I simply removed the sigmoid function from line 24 and pasted it in line 31.

Reply:
That's why the issue would have occurred! You shouldn't have removed it from the original function and pasted!